DuggaSys/diagram.php

```diff
@@ -256,27 +256,53 @@
                             <div class="drop-down-divider">
                             </div>
                             <div class="drop-down-item" tabindex="0">
-                                <span class="drop-down-option" onclick="toggleVirtualA4(event)">Display Virtual A4</span>
-                                <i id="hotkey-displayA4" class="hotKeys">Shift + 4</i>
-                            </div>
-                            <div class="drop-down-item" tabindex="0">
-                                <div id="a4-single-item" class="drop-down-item-disabled">
-                                    <span class="drop-down-option" onclick='togglesingleA4(event);'><img src="../Shared/icons/Arrow_down_right.png">Single A4</span>
+                                <span class="drop-down-option" onclick="toggleVirtualPaper(event)">Display Virtual Paper</span>
+                                <i id="hotkey-displayPaper" class="hotKeys">Shift + 4</i>
+                            </div>
```

Comment:
is there a reason for this indentation or is it just a visual bug on github?

Reply:
referring to row 262-287

Reply:
it seems like i don't have the right settings on indentations....

Reply:
it's better now